### PR TITLE
Add angular list builder and table column customization modal

### PR DESCRIPTION
**Pull request by Donisius**

### CODE REVIEW

**Review by zvonimirfras — CHANGES_REQUESTED** [review]

File: packages/angular/src/list-builder-index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export { ListBuilderModule } from './list-builder/list-builder.module';
+export { AIListBuilderComponent } from './list-builder/list-builder.component';
```
[the remark this answers is not in the document]
Comment (Donisius) [reply packages/angular/src/list-builder-index.ts:2]: I like keeping the AI for all components

File: packages/angular/src/list/list-item/ai-list-item.component.ts [hunk 2]
```diff
@@ -140,11 +141,21 @@ export class AIListItemComponent implements OnInit {
...
+  selectSubscription: Subscription;
+
   constructor(protected iconService: IconService) {}
 
   ngOnInit() {
     this.iconService.register(ChevronUp16);
     this.iconService.register(Draggable16);
+
+    this.selectSubscription = this.item.onSelect.subscribe(() => {
+      this.itemSelected.emit();
```
[the remark this answers is not in the document]
Comment (Donisius) [reply packages/angular/src/list/list-item/ai-list-item.component.ts:153]: yep

File: packages/angular/src/list-builder-index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (zvonimirfras) [reply packages/angular/src/list-builder-index.ts:2]: If we don’t have the AI* for the ones that don’t exist in cca, when they “graduate” and are moved to cca, we don’t have to change much, and people using it only change the import location.
If we’re adding th AI* only for the ones that already exist in cca, it’s clear that a component is a PAL variant

**Comment by davidicus** [comment]
@zvonimirfras @Donisius can we get an update to this or close it out? It is getting pretty stale.

**Comment by davidicus** [comment]
closing this from lack of response. Feel free to re-open when you are ready.

**Pull request closed without merging**